src/sqltest/platform/dialog/dialogPane.test.ts

```diff
@@ -90,38 +61,40 @@ suite('Dialog Pane Tests', () => {
 				validationCallbacks.push(params.validityChangedCallback);
 			});
 
+		let save = bootstrapAngular;
+
+		(<any>bootstrapAngular) = (collection, moduleType, container, selectorString, params: DialogComponentParams, input, callbackSetModule) => {
+			validationCallbacks.push(params.validityChangedCallback);
+		};
+
 		let modelViewId1 = 'test_content_1';
 		let modelViewId2 = 'test_content_2';
 		dialog.content = [new DialogTab('tab1', modelViewId1), new DialogTab('tab2', modelViewId2)];
 		let dialogPane = new DialogPane(dialog.title, dialog.content, valid => dialog.notifyValidityChanged(valid), mockInstantiationService.object);
 		dialogPane.createBody(container);
 
 		let validityChanges: boolean[] = [];
-		dialog.onValidityChanged(valid => validityChanges.push(valid));
 
+		dialog.onValidityChanged(valid => validityChanges.push(valid));
 		// If I set tab 2's validation to false
 		validationCallbacks[1](false);
-
 		// Then the whole dialog's validation is false
 		assert.equal(dialog.valid, false);
 		assert.equal(validityChanges.length, 1);
 		assert.equal(validityChanges[0], false);
-
 		// If I then set it back to true
 		validationCallbacks[1](true);
-
 		// Then the whole dialog's validation is true
 		assert.equal(dialog.valid, true);
 		assert.equal(validityChanges.length, 2);
 		assert.equal(validityChanges[1], true);
-
 		// If I set tab 1's validation to false
 		validationCallbacks[0](false);
-
 		// Then the whole dialog's validation is false
 		assert.equal(dialog.valid, false);
 		assert.equal(validityChanges.length, 3);
 		assert.equal(validityChanges[2], false);
 
+		(<any>bootstrapAngular) = save;
```

Comment:
This won't execute if one of the asserts fails. Could that cause problems for other tests? Would it work if this were in the `teardown` function and the code setting it up were in the `setup` function for this test suite?